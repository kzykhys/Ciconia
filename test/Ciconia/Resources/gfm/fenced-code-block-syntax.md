

    code block
    code block

``` php
<?php

class Test
{

}
```

header
------

1. foo

    ```
    Code
    ```

2. bar
