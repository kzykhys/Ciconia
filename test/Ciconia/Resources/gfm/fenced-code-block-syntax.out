<pre><code>code block
code block
</code></pre>

<pre class="prettyprint lang-php"><code class="language-php">&lt;?php

class Test
{

}
</code></pre>

<h2>header</h2>